mkdocs.yml

```diff
@@ -69,5 +69,7 @@ nav:
     - GAMMA: contributing/gamma.md
   - Blog: 
     - Index: blog/index.md
+    - 2022:
+      - blog/2022/first-beta.md
```

Comment:
URL path `blog/2022/first-beta.md` ≭ repo path `site-src/blog/2022/graduating-to-beta.md`

Reply:
Updated, thanks!